packages/itwin/ec3-widget/src/components/api/context/ApiConfigContext.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import type { AccessToken } from "@itwin/core-bentley";
+import * as React from "react";
+import { createContext } from "react";
+
+export type ClientPrefix = "" | "dev" | "qa" | undefined;
```

Comment:
Instead of prefixes, we should allow the base URL for both carbon calculation and reporting to be passed in as props instead. Similar to Reports Config widget or how the `ec3Uri` can be passed in as well. G&M widget is an exception due to a dependency only allowing prefixes.

Reply:
Changed props to accept base path and not prefix. (this is handled in ```EC3Config.ts``` file)